src/main/webapp/app/modules/surveyCreator/surveyCreator.tsx

```diff
@@ -158,13 +276,26 @@ const SurveyCreator = () => {
               Add Question
             </Button>
           </Col>
+          <Col className="d-flex justify-content-center">
+            <Button
+              color="primary"
+              id="save-entity"
+              data-cy="entityCreateSaveButton"
+              type="submit"
+              style={{ borderRadius: '25px' }}
+              onClick={handleClick}
+            >
+              Create survey
```

Comment:
przydałoby się redirectować po stworzeniu do widoku tej survey, bo po kliknięciu oprócz stworzenia nic się nie dzieje

Reply:
Tutaj byłby do zrobienia nowy widok, żeby ankieta wyświetlała się wraz z pytaniami, natomiast aktualnie baza nie zwraca pytań dla danej ankiety, więc nie wiem czy jest sens robić to teraz?